x-pack/plugin/sql/sql-action/src/test/java/org/elasticsearch/xpack/sql/action/SqlQueryRequestTests.java

```diff
@@ -172,7 +172,7 @@ private RequestInfo randomRequestInfo() {
     }
 
     private TimeValue randomTV() {
-        return TimeValue.parseTimeValue(randomTimeValue(), null, "test");
+        return randomTimeValue();
```

Comment:
This method does not seem to be useful after this refactor

Reply:
Good point, see 45333d2e6f2528e44238f61d13aaf1da483b1de8.